sysID/fit_flywheel_arms_philip.py

```diff
@@ -0,0 +1,193 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from models import cartpole_arm_flywheel as model
+
+# Load the data
+q = np.load('/Users/philipqueen/Documents/Humon Research Lab/FreeMocap_Data/4stepsequence_session2_10_5_22/pendulum_flywheel_arms_state_q.npy')
+qDot = np.load('/Users/philipqueen/Documents/Humon Research Lab/FreeMocap_Data/4stepsequence_session2_10_5_22/pendulum_flywheel_arms_state_qdot.npy')
+qDotDot = np.load('/Users/philipqueen/Documents/Humon Research Lab/FreeMocap_Data/4stepsequence_session2_10_5_22/pendulum_flywheel_arms_state_qddot.npy')
+
+params = model.default_params()
+
+# convert to meters
+q[:, 0] = q[:, 0]/1000.
+q[:, 3] = q[:, 3]/1000.
```

Comment:
See the last image I posted on June 26th in slack: the order of the variables in the model are [cartpole position, pendulum angle, ARM POSITION, flywheel angle].
So this should be q[:, 2] (if I understood this correctly)

Reply:
I fixed the positions to match the ordering you mentioned